rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/WebClient.java

```diff
@@ -90,11 +90,19 @@
     private static final String WEB_CLIENT_OPERATION_REPORTING = "enable.webclient.operation.reporting";
     private BodyWriter bodyWriter = new BodyWriter();
     protected WebClient(String baseAddress) {
-        this(convertStringToURI(baseAddress));
+        this(convertStringToURI(baseAddress), Collections.emptyMap());
```

Comment:
Same comment as before wrt Collections.emptyMap() - it would probably perform better if it were a constant. Same for lines 101 and 121.